hyperion-shared-preferences/src/main/java/com/willowtreeapps/hyperion/sharedpreferences/SharedPreferencesViewerActivity.java

```diff
@@ -0,0 +1,15 @@
+package com.willowtreeapps.hyperion.sharedpreferences;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+public class SharedPreferencesViewerActivity extends AppCompatActivity {
```

Comment:
Probably want to add `@HyperionIgnore` to this Activity

Reply:
Ah, I must have missed that when I was reviewing the other plugins.